backend/overstroomik_service/config.py

```diff
@@ -9,6 +9,15 @@ class Settings(BaseSettings):
     # "http://localhost:8080", "http://local.dockertoolbox.tiangolo.com"]'
     BACKEND_CORS_ORIGINS: List[AnyHttpUrl] = []
     PORT: int = 8000
+    PDOK_API = "https://geodata.nationaalgeoregister.nl/locatieserver/v3/free"
+    WS_VERSION = 0.1
+
+    ERROR_GENERAL_NOER = "no-error"
```

Comment:
I like the error constants, but they don't need to be in the settings class. This settings class is for autoparsing .env files and ENV variables. PDOK_API is a good thing to put in here, so we can change it with only changing the .env file.

Reply:
Constants removed from config